Add EOL statement to README (#195)
cloudant/swift-cloudant: README.md

# DEPRECATED

> **This library is now deprecated and not supported by IBM.**

Reviewer: I don't think we need to say IBM, it isn't supported by anyone in this repo.
```suggestion
**This library is end-of-life and no longer supported.**
```

> However, GigabiteLabs dedicated to continue this work. 
> Please visit their forked version with your concern: https://github.com/GigabiteLabs/swift-cloudant-lts

Reviewer: I think we should be more vague here, the `README` in the archived repo might long outlast any given fork and new forks may emerge.
```suggestion
Please see the list of [forks](https://github.com/cloudant/swift-cloudant/network/members) for continuing development efforts (e.g. #194).
```

Reviewer 2: Changed in 5e0f275

Reviewer 2: Also removed the first sentence: 698734b
